packages/web-components/src/components/footer/defs.ts

```diff
@@ -27,6 +29,8 @@ export enum FOOTER_SIZE {
   MICRO = 'micro',
 }
 
+// TODO: Remove when JAWS bug has been fixed
```

Comment:
can this TODO be removed now?

Reply:
I added that; those new lines come from the `carbon-web-components`'s defs file. Only added as a note so whenever the bug gets fixed in that repo we can remove all the additions from this PR, including the stuff here in `defs`